src/main/java/org/spongepowered/common/mixin/core/util/math/Vec3iMixin.java

```diff
@@ -43,7 +43,7 @@
 
     @Override
     public boolean bridge$isValidPosition() {
-        return this.x >= -30000000 && this.z >= -30000000 && this.x < 30000000 && this.z < 30000000 && this.y >= 0 && this.y < 256;
+        return this.x >= -30000000 && this.z >= -30000000 && this.x < 30000000 && this.z < 30000000 && this.y >= 0 && this.y < 1024;
```

Comment:
I understand the below change but I don't think this change is correct. A valid position should be within Minecraft's typical bounds.

Reply:
How about leaving this method as-is and have the code using it call `isValidXZPosition()` instead?